openapi/components/schemas/PayoutRequest.yaml

```
@@ -93,6 +93,13 @@ properties:
             - $ref: ./ServerTimestamp.yaml
         updatedTime:
           $ref: ./UpdatedTime.yaml
+  selectedPaymentInstrumentRedirectUrl:
+    description: |-
+      URL to redirect the customer to when payment instrument is selected. The default value is the website URL.
```

suggestion: I think this helps make it more concise
```suggestion
      URL where the customer is redirected when a payment instrument is selected. The default value is the website URL.
```